mod/metrics/spec/set/type_plus_right/metric/all_metric_values/stats_spec.rb

```diff
@@ -10,8 +10,8 @@
         with_tag :tr, count: 4
         with_row "", :known, 11, "Known"
         with_row "+", :unknown, 2, "Unknown"
-        with_row "+", :none, 16, "Not Researched"
-        with_row "=", :total, "29", "Total results"
+        with_row "+", :none, 14, "Not Researched"
```

Comment:
These changed after I re-migrated test data and I have no idea why.  I'm more uncomfortable with this change than any other.